src/pages/ai.tsx

```diff
@@ -43,7 +43,7 @@ export default function AiPage() {
                 <p className="text-xl">{heroSection.text}</p>
                 <div>
                   <Link
-                    className="rounded-full bg-gradient-to-b from-green-500 to-blue-500 px-10 py-6 text-2xl font-medium lg:py-4 lg:px-8 lg:text-lg"
+                    className="rounded-full bg-gradient-to-b from-green-500 to-blue-500 px-10 py-6 text-2xl font-medium lg:py-4 lg:px-8 lg:text-lg text-sm"
```

Comment:
there's a css conflict here, `text-sm` and `text-2xl` are set. Do you get a css conflict warning in your workspace? 
Also as a side note, it looks like the auto class sorting with prettier isn't taking effect. For example, `text-sm` remains last in the class names on LN46 in `ai.tsx`. It should be where `text-2xl` is if auto class sorting worked.

Reply:
where would I see this error?

Reply:
hmm I dont see that

Reply:
it is fixed on current push

Reply:
is your prettier tailwind plugin working?

Reply:
do you mean extension?

Reply:
I think you need both prettier tailwind plugin and tailwind css intellisense enabled in order for tailwind linting and auto class sorting to properly work

Reply:
thank capt. do you mean Tailwind class sorter? I cant find the prettier tailwind plugin

Reply:
Slack me to discuss further